Project/Engine/src/sysFrame/Object/Camera/Camera.cpp

```
@@ -2,20 +2,11 @@
-    UUID uuid;
-    UuidCreate(&uuid);
-    RPC_CSTR szUuid = nullptr;
-    UuidToStringA(&uuid, &szUuid);
-    struct UUIDCleaner{
-        RPC_CSTR& ptr;
-        ~UUIDCleaner() {
-            if (ptr)RpcStringFreeA(&ptr);
-        }
-    } cleaner {szUuid};
-    uuid_ = reinterpret_cast<char*>(szUuid);
+    uuid_ = System::CreateUuid();
```

Cppcheck (2.10-2)

[performance] Variable 'uuid_' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)